### PR TITLE
build(all): configure TS to use paths instead of project references

### PR DESCRIPTION
**Pull request by clebert**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,14 +6,15 @@
     "packages/*"
   ],
   "scripts": {
-    "prebuild": "rimraf docs && lerna exec --parallel -- rimraf lib",
-    "build": "yarn build:sources && lerna exec --parallel -- typedoc --options ../../typedoc.js --out ../../docs/api/\\$LERNA_PACKAGE_NAME --readme README.md --tsconfig tsconfig.typedoc.json .",
+    "prebuild": "rimraf docs",
```
Comment (KingHenne) [inline package.json]: Deleting `lib` before building is still valid, even without project references, isn't it?

Comment (clebert) [reply package.json]: we delete it now before build:sources

Comment (clebert) [reply package.json]: before we didn't do that because we broke the caching. that doesn't exist anymore, does it?

Comment (KingHenne) [reply package.json]: ah, I missed that

**Pull request merged**